index.html

```diff
@@ -114,31 +146,65 @@
           <dfn>supportedNetworks</dfn>
         </dt>
         <dd>
-          The <a>supportedNetworks</a> member contains a sequence of
-          identifiers for card networks that the merchant accepts. W3C
-          maintains a <a href=
+          A sequence of identifiers for card networks that the merchant
+          accepts. W3C maintains a <a href=
           "https://www.w3.org/Payments/card-network-ids">list of approved card
           network identifiers</a>.
         </dd>
         <dt>
           <dfn>supportedTypes</dfn>
         </dt>
         <dd>
-          The <a>supportedTypes</a> member contains a sequence of card types
-          that the merchant accepts. Implementations will determine how to
-          match the type values <span data-link-for=
-          "BasicCardType"><a>credit</a>, <a>debit</a>, and
-          <a>prepaid</a></span>.
+          A sequence of card <a>types</a> that the merchant accepts.
         </dd>
       </dl>
-      <p class="note">
-        The <a>supportedNetworks</a> and <a>supportedTypes</a> members are both
-        optional. If neither is provided then any card may be returned. If only
-        <a>supportedNetworks</a> is provided then any card type may be returned
-        provided it matches one of the networks. If only <a>supportedTypes</a>
-        is provided then a card may be returned from any network provided it
-        matches one of the types.
+    </section>
+    <section data-link-for="BasicCardRequest">
+      <h2>
+        Interfacing with a payment handler
+      </h2>
+      <p>
+        The <dfn>steps to constrain a payment handler</dfn> with
+        <var>data</var> <a>BasicCardRequest</a> are given by the following
+        algorithm. If the end-user inputs or selects a <a>card</a> that meets
```

Comment:
algorithm. If the end-user inputs or selects a <a>card</a> that meets

“end-user” doesn’t need a hyphen; should just be “end user”